core/src/main/java/com/linecorp/armeria/common/ContentTooLargeException.java

```diff
@@ -25,18 +25,48 @@
 
     private static final ContentTooLargeException INSTANCE = new ContentTooLargeException(false);
 
+    private long transferred;
+    private long total;
+    private long maximum;
+
     /**
-     * Returns a {@link ContentTooLargeException} which may be a singleton or a new instance, depending on
-     * {@link Flags#verboseExceptionSampler()}'s decision.
+     * Returns a {@link ContentTooLargeExceptionBuilder} which may return a singleton or a new instance in its
+     * {@code build} method, depending on {@link Flags#verboseExceptionSampler()}'s decision.
      */
-    public static ContentTooLargeException get() {
```

Comment:
We cannot remove this method because it's a breaking change. Let's bring back this method.
We need to deprecate and add TODO to remove it when we reach 2.0.